lib/tlIO/STBRead.cpp

```diff
@@ -0,0 +1,202 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2021-2023 Darby Johnston
+// All rights reserved.
+
+#define STBI_NO_JPEG
+#define STBI_NO_PNG
+#define STBI_NO_HDR
+#define STBI_NO_PNM
+#define STB_IMAGE_IMPLEMENTATION
+#include <stb/stb_image.h>
+
+#include <tlIO/STB.h>
+
+#include <tlCore/StringFormat.h>
+
+namespace tl
+{
+    namespace stb
+    {
+        namespace
+        {
+            
+            class File
+            {
+            public:
+                File(const std::string& fileName, const file::MemoryRead* memory)
+                    {
```

Comment:
It looks like this block of code got extra indentation, the opening and closing brackets are not aligned.

Reply:
Fixed.